core/src/main/java/org/springframework/security/access/intercept/AbstractSecurityInterceptor.java

```diff
@@ -201,17 +212,8 @@ protected InterceptorStatusToken beforeInvocation(Object object) {
 					"An Authentication object was not found in the SecurityContext"), object, attributes);
 		}
 		Authentication authenticated = authenticateIfRequired();
-		if (this.logger.isTraceEnabled()) {
-			this.logger.trace(LogMessage.format("Authorizing %s with attributes %s", object, attributes));
-		}
 		// Attempt authorization
-		attemptAuthorization(object, attributes, authenticated);
-		if (this.logger.isDebugEnabled()) {
-			this.logger.debug(LogMessage.format("Authorized %s with attributes %s", object, attributes));
-		}
-		if (this.publishAuthorizationSuccess) {
-			publishEvent(new AuthorizedEvent(object, attributes, authenticated));
```

Comment:
I wonder if `FilterInvocationAuthorizationManager` should fire an `AuthorizedEvent`. I'm thinking out loud here just a bit, but the important thing to remember is that upgrading to the latest shouldn't mean your events stop firing.